.github/workflows/flow-deploy-release-artifact.yaml

```diff
@@ -150,7 +150,7 @@ jobs:
           GRADLE_PUBLISH_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
           GRADLE_PUBLISH_SECRET: ${{ secrets.GRADLE_PUBLISH_SECRET }}
         run: |
-          ./gradlew release-maven-central --no-parallel --scan -PpublishSigningEnabled=true \
+          ./gradlew releaseMavenCentral --no-parallel --scan -PpublishSigningEnabled=true \
```

Comment:
Should we rename this task everywhere to just `release`? The current name is confusing IMO as it releases to the Gradle Plugin Portal as well.

Reply:
I have no issues renaming to `release` and `releaseSnapshot` as long as `release` does not cause any issues.

Reply:
I did the rename. Feel free to take over again and merge.